### PR TITLE
Add remove item into cart

### PR DESCRIPTION
**Pull request by ElenaQQQ**
Add the second test for cart functionality, add git ignore file

### CODE REVIEW

**Comment by YaninaProkharava** [comment]
Reviewed your code, everything looks good!

**Review by YaninaProkharava — COMMENTED** [review]
good!

**Review by rahim1sa — COMMENTED** [review]
Reviewed!

File: src/test/java/org/example/AddRemoveItemToCartTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//span[@class='a-size-medium-plus a-color-base sw-atc-text a-text-bold']")));
+        Assert.assertTrue(addedToCard.isDisplayed());
+
+        WebElement closeSideSheet = new WebDriverWait(webDriver,Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.id("attach-close_sideSheet-link")));
+        closeSideSheet.click();
+
+        WebElement cartAmount = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.id("nav-cart-count")));
+        Assert.assertEquals("1", cartAmount.getText());
```
Comment (juliaHrabovska) [inline src/test/java/org/example/AddRemoveItemToCartTest.java]: It is better to add a string that explains why the test failed. Same for other

Comment (ElenaQQQ) [reply src/test/java/org/example/AddRemoveItemToCartTest.java]: Fixed

Comment (ElenaQQQ) [reply src/test/java/org/example/AddRemoveItemToCartTest.java]: @juliaHrabovska fixed, available to merge

**Comment by juliaHrabovska** [comment]
Everything is good, but can not merge, because of conflicts. Can you fix them and after that you can merge :)

**Pull request merged**